app/models/concerns/api_data.rb

```diff
@@ -0,0 +1,9 @@
+module ApiData
+  extend ActiveSupport::Concern
+
+  def api_data
+    self.all.map do |s|
+      (Api::SimpleSerializer.new(s)).as_json(root: false)
+    end
```

Comment:
@marcellosachs The serializers should be part of the presentation layer, so it seems kind of funky to mix them into the models as a concern. I get what you're going for here, though I'd probably pull this `api_data` method out into a separate object that just mediated between an arbitrary model and the serializers.